src/public/modules/forms/admin/views/list-forms.client.view.html

```diff
@@ -6,11 +6,6 @@
     <div id="list-form" class="row" ng-if="vm.user">
       <!-- When forms have already been created -->
       <div id="list-form-exist" ng-if="vm.myforms.length > 0">
-        <div
```

Comment:
how do the form duplicate/delete buttons get hidden now that this div is deleted?

Reply:
oh my god, that's what they are for???? I'll add them back. Thanks for the catch LOL. Thought they used some clickoutside directive..............